tests/System.Text.Http.Tests/GivenAnHttpHeaders.cs

```diff
@@ -25,46 +25,32 @@ public class GivenAnHttpHeaders
 
         private const string HeaderWithoutCrlf = "Host: localhost:8080";
 
-        private HttpHeadersSingleSegment _httpHeaders;
-
-        public GivenAnHttpHeaders()
-        {
-            Span<byte> headers;
-            var bytes = new UTF8Encoding().GetBytes(HeadersString);
-
-            unsafe
-            {
-                fixed (byte* buffer = bytes)
-                {
-                    headers = new Span<byte>(buffer, bytes.Length);
-                }
-            }
-
-            _httpHeaders = new HttpHeadersSingleSegment(headers);
-        }
-
-        [Fact]
+        [Fact(Skip = "System.TypeLoadException : The generic type 'System.Collections.Generic.KeyValuePair`2' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
```

Comment:
Utf8String implements IEnumerable as well. I did not see issues there.

https://github.com/dotnet/corefxlab/blob/master/src/System.Text.Primitives/System/Text/Encoding/Utf8/Utf8String.CodePointEnumerable.cs#L11